src/routes/register/+page.svelte

```
@@ -75,13 +72,15 @@
 			return;
 		}
 
-		const response = await fetch('api/account/username/exists?username=' + username);
+		const response: Response = await fetch(
+			'api/account/username/exists?username=' + username.trim()
+		);
 		if (!response.ok) {
 			usernameMessage = 'Fehler beim überprüfen des Namen. Fehlercode: ' + response.status;
```

I don't think we should show this to the user. Maybe the rate limiting (not yet implemented) has just rejected the request. Showing this to the user can be confusing. Maybe just ignore the error?

---

done